docs/content/installation/install_compatibility_requirements.md

```diff
@@ -51,13 +51,20 @@ Complete these steps for each worker node in Kubernetes cluster to prepare your
 
    `oc apply -f 99-ibm-attach.yaml`
 
-2. Configure storage system connectivity.
+2. Configure your storage system host attachment.
 
-    1.  Define the host of each Kubernetes node on the relevant storage systems with the valid WWPN (for Fibre Channel) or IQN (for iSCSI) of the node.
-
-    2.  For Fibre Channel, configure the relevant zoning from the storage to the host.
+    **Important:** The CSI driver does not define hosts on your storage system.
+    
+    Be sure to configure your storage system host attachment according to your storage system instructions.
 
-    3. Ensure proper connectivity.
+    The CSI driver supports the following connectivity for each worker node: Fibre Channel (WWPN), NVMe®/FC (NQN), and iSCSI (IQN).
+        
+    **Note:** 
+        - The CSI driver only supports NVMe®/FC on IBM Spectrum Virtualize family products.
+        - For Fibre Channel connectivity be sure that storage system is using one of the fully supported HBAs compatible with your host connection, as listed in the [IBM® System Storage® Interoperation Center (SSIC)](https://www-03.ibm.com/systems/support/storage/ssic/interoperability.wss).
+        - IBM DS8000 family storage systems only supports Fibre Channel connectivity.
```

Comment:
```suggestion
        - The CSI driver supports the IBM DS8000 family storage systems only with Fibre Channel connectivity.
```
(so our docs remain true regardless of ds8k changes)

Reply:
@erantzabari ?

Reply:
I agree with Ori here Rivka

Reply:
done